support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -237,7 +237,7 @@ const isCardUserSelected = (
 	);
 };
 
-const productCatalogDescInclOffers = {
+const productCatalogDescInclOffers: typeof productCatalogDescExclOffers = {
```

Comment:
Do you need to align the types? Presume thats why `benefits: []` needed to be added back to ProductCatalog?